pypika/queries.py

```diff
@@ -148,6 +148,15 @@ def __ne__(self, other):
     def __hash__(self):
         return hash(str(self))
 
+    def select(self, *terms):
+        return Query.from_(self._table_name).select(*terms)
```

Comment:
Need to pass in the instance of Table to `from_` in order to carry all of the information over. Can be just changed to `Query.from_(self) ... `

Reply:
@twheys Sure. That makes sense. Let me change that